src/recordlinker/routes/person_router.py

```diff
@@ -170,3 +170,39 @@ def merge_person_clusters(
         service.delete_persons(session, persons, commit=False)
 
     return schemas.PersonRef(person_reference_id=person.reference_id)
+
+
+@router.delete(
+    "/{person_reference_id}",
+    summary="Delete an empty Person",
+    status_code=fastapi.status.HTTP_204_NO_CONTENT,
+)
+def delete_empty_person(
+    person_reference_id: uuid.UUID,
+    session: orm.Session = fastapi.Depends(get_session),
+):
+    """
+    Delete an empty Person from the MPI database.
+    """
+    # Check that person_reference_id is valid
+    person = service.get_person_by_reference_id(session, person_reference_id)
+
+    if person is None:
+        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_404_NOT_FOUND)
+
+    # Check if the person has associated patients
+    if person.patients:
+        raise fastapi.HTTPException(
+            status_code=fastapi.status.HTTP_403_FORBIDDEN,
```

Comment:
Thoughts on using a 422 here instead of a 403.  I think redoc automatically creates documentation for 422 when we use pydantic for validation.

Reply:
I checked, and it just shows the 204 and 422.  We can also update the docs and add a 3rd response type.

<img width="1379" alt="Screenshot 2025-02-20 at 16 23 33" src="https://github.com/user-attachments/assets/fe91a9ef-12b6-48c2-b661-7cca6c06cc96" />

Reply:
I opted to add the docs for 403 since I think that error will make more sense than getting a 422 in this instance. Also allows us more flexibility moving forward. Let me know if you think `ErrorResponse` and `ErrorDetail` should be in a different schema file (I stuck them in mpi)